R/telemetry.R

```diff
@@ -713,7 +716,11 @@ Telemetry <- R6::R6Class( # nolint object_name_linter
       }
     },
 
-    get_user = function(session = shiny::getDefaultReactiveDomain()) {
+    get_user = function(
+      session = shiny::getDefaultReactiveDomain(),
+      username = NULL
+    ) {
+      if (!is.null(username)) return(username)
```

Comment:
(optional) Given that there's no documentation for the function, I think that the parameter name would benefit from being a bit more verbose.

```suggestion
    get_user = function(
      session = shiny::getDefaultReactiveDomain(),
      force_username = NULL
    ) {
      if (!is.null(force_username)) return(force_username)
```